gc/base/standard/Scavenger.cpp

```diff
@@ -248,6 +248,9 @@ MM_Scavenger::initialize(MM_EnvironmentBase *env)
 	case MM_GCExtensionsBase::OMR_GC_SCAVENGER_SCANORDERING_BREADTH_FIRST:
 		_cachesPerThread = FLIP_TENURE_LARGE_SCAN;
 		break;
```

Comment:
you can remove these two lines and let first case 'falls through' the second case